static/js/fhir.js

```diff
@@ -0,0 +1,50 @@
+let fhir_url = "https://api-dev.oninnovaccer.com/fhir/";
+let resource, param, page;
+let pagination_size = 5;
+
+//Function for hitting the API with the query parameters
+//and returning the json object
+async function FHIRSearch(resource, param, page) {
+  let myHeaders = new Headers();
+  let application_token = storageDriver(
```

Comment:
should be camel case as `applicationToken`

Reply:
also, shouldn't you be using @RhnSaxena oauth lib to get application token instead of fetching it from storage ?